### PR TITLE
models and mako for hochwasserstatistik

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

File: chsdi/locale/en/LC_MESSAGES/chsdi.po [hunk 1]
```diff
@@ -1086,19 +1086,19 @@ msgid "ch.bafu.bundesinventare-amphibien_anhang4"
 msgstr "Amphibians appendix 4"
 
 msgid "ch.bafu.bundesinventare-amphibien_anhoerung"
-msgstr "Amphibien - Anhörung 2015"
```
Comment (cedricmoullet) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: Why do we remove these translations ?

File: chsdi/locale/fr/LC_MESSAGES/chsdi.po [hunk 2]
```diff
@@ -1941,7 +2004,7 @@ msgid "ch.blw.feldblockkarte"
 msgstr "Carte de blocs de parcelles"
 
 msgid "ch.blw.gewaesseranschlusskarte"
-msgstr "Surface en connexion des eaux"
+msgstr "Surface en connection des eaux"
```
Comment (cedricmoullet) [inline chsdi/locale/fr/LC_MESSAGES/chsdi.po]: connexion et pas connection

File: chsdi/locale/en/LC_MESSAGES/chsdi.po (hunk 1 above)
Comment (loicgasser) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: Yeah we lost it for some reason, same in my geojson PR. Simply put it back in the translation table.

**Comment by pfanguin** [comment]
In the translations table, the field was filled by two single quote. I don't know why.

File: chsdi/templates/htmlpopup/hochwasserstatistik.mako [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    else :
+        url = 'url_de' 
+%>
+    <tr>
+      <td class="cell-left">${_('ch.bafu.hydrologie-hochwasserstatistik.name')}</td>
+      <td>${c['attributes']['name'] or '-'}</td>
+    </tr>
+    <tr>
+      <td class="cell-left">${_('ch.bafu.hydrologie-hochwasserstatistik.einzugsgebietsflaeche')}</td>
+      <td>${c['attributes']['einzugsgebietsflaeche']}</td>
```
Comment (gjn) [inline chsdi/templates/htmlpopup/hochwasserstatistik.mako]: Maybe also use the `or` mechanism here?

File: chsdi/templates/htmlpopup/hochwasserstatistik.mako [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    <td class="cell-meta">${_('ch.bafu.hydrologie-hochwasserstatistik.coordX')}</td>
+    <td class="cell-meta">${c['attributes']['x_koord'] or '-'}</td>
+  </tr>
+  <tr>
+    <td class="cell-meta">${_('ch.bafu.hydrologie-hochwasserstatistik.coordY')}</td>
+    <td class="cell-meta">${c['attributes']['y_koord'] or '-'}</td>
+  </tr>
+  <tr>
+    <td class="cell-meta">${_('ch.bafu.hydrologie-hochwasserstatistik.einzugsgebietsflaeche')}</td>
+    <td class="cell-meta">${c['attributes']['einzugsgebietsflaeche']}</td>
```
Comment (gjn) [inline chsdi/templates/htmlpopup/hochwasserstatistik.mako]: dito

**Comment by gjn** [comment]
Added 2 small comment. Please fix and merge.

**Comment by loicgasser** [comment]
rebase

**Comment by pfanguin** [comment]
It's done.

**Comment by pfanguin** [comment]
rebase done!

**Comment by loicgasser** [comment]
when is this supposed to go to prod?

**Comment by pfanguin** [comment]
for July

**Pull request merged**